sdk/core/azure-json/src/main/java/com/azure/json/implementation/jackson/core/Base64Variant.java

```diff
@@ -0,0 +1,886 @@
+/* Jackson JSON-processor.
```

Comment:
we need to make sure to comply with [apache license](https://github.com/FasterXML/jackson-core/blob/2.14/LICENSE)

https://fossa.com/blog/open-source-licenses-101-apache-license-2-0/